src/Option.ts

```diff
@@ -204,6 +225,27 @@ export class Some<A>
   toString() {
     return `some(${toString(this.value)})`
   }
+  contains(setoid: Setoid<A>, a: A): boolean {
```

Comment:
Ah yes, I see how it mirrors `equals`. I don't like how that breaks compatibility with most other libraries. Is there a reason for that @gcanti?